tests/config/test_config.py

```diff
@@ -20,24 +19,9 @@
     from pathlib import Path
 
 
-def flatten(config: Optional[Dict[str, Any]], parent_key: str = "", sep="."):
-    items = []
-    for k, v in config.items():
-        new_key = parent_key + sep + k if parent_key else k
-        if isinstance(v, dict):
-            items.extend(flatten(v, new_key, sep=sep).items())
-        else:
-            items.append((new_key, v))
-    return dict(items)
-
-
-def get_options_based_on_normalizer(
-    normalizer: Callable, config: Optional[Config] = None
-) -> str:
-    if config is None:
-        config = Config()
-
-    flattened_config = flatten(config.default_config)
+def get_options_based_on_normalizer(normalizer: Callable) -> str:
+    config = Config()
+    flattened_config = FlatDict(config.default_config, delimiter=".")
```

Comment:
...in favor of this.

```suggestion
    flattened_config = FlatDict(Config.default_config, delimiter=".")
```